### PR TITLE
AnimatedLink component added

### PR DESCRIPTION
**Pull request by peratik**
https://www.notion.so/santiment/Link-buttons-updates-ddb3cd3c912341a880e46a17b302bd35

image

### CODE REVIEW

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]
1. There is `btn-0` class for highlighting action elements;
2. Action with a pointer should be a component

File: src/ui/LinkPointer.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  let className = ''
+  export { className as class }
+  export let href: string
+  export let label: string
+</script>
+
+<a {...$$props} {href} class="link-pointer v-center {className}">
+  {label}
+  <Svg id="right-arrow" w={10} h={5} class="$style.svg" />
```
Comment (peratik) [inline src/ui/LinkPointer.svelte:12]: Not all links have right-arrow

File: stories/LinkPointer.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  import Link from '@/ui/LinkPointer.svelte'
+</script>
+
+<Meta title="Example/LinkPointer" />
+
+<Story name="LinkPointer">
+  <div class="body-3">
+    <a href="/" class="link-pointer">body-3</a>
+    <br />
+    <Link href="/" label="body-3 + pointer" />
```
Comment (peratik) [inline stories/LinkPointer.stories.svelte:12]: Watch this in the browser:
hover line starts thin and at the end becomes thick

Comment (peratik) [reply stories/LinkPointer.stories.svelte:12]: Start:
image

End:
image

File: src/ui/LinkPointer.svelte (hunk 1 above)
Comment (peratik) [reply src/ui/LinkPointer.svelte:12]: Task has 2 parts:
https://www.notion.so/santiment/Link-buttons-updates-ddb3cd3c912341a880e46a17b302bd35

Comment (DmitriVanGuard) [reply src/ui/LinkPointer.svelte:12]: See `stories/LinkPointer.stories.svelte`

File: stories/LinkPointer.stories.svelte (hunk 2 above)
Comment (DmitriVanGuard) [reply stories/LinkPointer.stories.svelte:12]: Tested in browserstack on multiple browsers/platforms. Looks like a local issue. Make sure there is no zoom or other rendering changes

**Pull request merged**